README.md

```diff
@@ -2942,3 +3020,23 @@ IMetadateDelete {
 ```
 
 This method deletes existing metadata for an asset or entry. It accepts metadata configuration as required arguments. This config contains basic details that you need to identify the metadata object you want to delete.
+
+## Modal
+
+The `Modal` class represents a modal dialog opened from the app within the Contentstack UI. It is a powerful feature of the App SDK that enables apps to open modal dialogues, providing an enhanced user experience.
```

Comment:
"This feature of the App SDK enables apps to open modal dialogues, providing an enhanced user experience." in place of "It is a powerful feature of the App SDK that enables apps to open modal dialogues, providing an enhanced user experience."

Reply:
Addressed.